picard/cluster.py

```diff
@@ -332,6 +345,10 @@ def can_view_info(self):
     def can_remove(self):
         return len(self.files) > 0
 
+    @property
+    def can_show_coverart(self):
```

Comment:
Perhaps move this to Item class instead.

Reply:
Done in 8dba1fd7d9e86eff2b187975db83d53125960a65. I would also like to make all those `can_*` functions to properties, but this is probably something for a separate PR